### ISSUE TITLE
What do you think is the best entry point to JS community?

### ISSUE THREAD

**Issue by maxkolyanov** [paulmillr/ama#5]
Lately, I find myself wanting to create something useful, contribute to the community.
I was lurking around different repos, organizations, reading source code and stuff for the last week, but still can't find a decent way to approach any of them. Either there's no open issues that suit me, or issues are too complex to make decisions alone and require discussing them with the community.

**Comment by paulmillr** [comment]
That's true, many of them require clarifications and discussions. I see no problem in that though! Just have a friendly conversation w maintainers and they'll most likely be able to help you!
